tests/security/cc/remove_bridge_network.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Remove bridge network in CC system role for multi-machine test
+# Maintainer: Liu Xiaojing <xiaojing.liu@suse.com>
+# Tags: poo#102116
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    my $bridge_info = script_output('bridge link');
+    if ($bridge_info =~ /\d+:\s+(eth\d):.*master\s(br\d).*/) {
```

Comment:
xiaojing,
Please do not use eth here. the name may change as udev rules [emx enpx etc]

Reply:
Thanks, fixed.